microsoft/testsuites/cpu/scripts/fio_network_client_workload.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+targetIPAddress=$1
```

Comment:
What's the reason, it needs a script? The iperf3 script can be started from LISA.

Reply:
`iperf3` needs to be run in a loop asynchronously. Currently in Lisa, I don't think there is a way to run this entire script with loop asynchronously. We can only run the statement "iperf3 -c $targetIPAddress -t 60 -P 32 >> workload.json" asynchronously